swap_meet/item.py

```diff
@@ -1,2 +1,12 @@
 class Item:
-    pass
+    def __init__(self, category = "", condition=0):
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        return "Hello World!"
+    
+    def condition_description(self):
+        descriptions = ["Trash","Used","Gently Used","Like New","New", "New Tags"]
+        return descriptions[self.condition]
```

Comment:
Just like we talked about in class, how would you handle floats?

Reply:
return descriptions[round(self.condition)] :)